src/Package/MSTest.TestFramework.nuspec

```diff
@@ -27,40 +27,6 @@
     <tags>MSTest TestFramework Unittest MSTestV2 Microsoft Test Testing TDD Framework </tags>
   </metadata>
   <files>
-    <!--DotNet-->
-    <file src="MSTest.Core\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="lib\dotnet" />
-    <file src="MSTest.Core\Microsoft.VisualStudio.TestPlatform.TestFramework.xml" target="lib\dotnet" />
```

Comment:
This should ideally be netstandard1.0. So extension writers can build against the lowest version. Please file an issue for this.